packages/nimble-components/src/anchor/index.ts

```diff
@@ -1,4 +1,4 @@
-import { attr } from '@microsoft/fast-element';
+import { attr, observable } from '@microsoft/fast-element';
```

Comment:
I'm not able to get row selection to work. From the HLD:
>If a row is focused, pressing Space will select/unselect that row 

When I focus an entire row, space either does nothing or scrolls the page. I tried this with both single select and multi select mode on the hierarchical table (again in macOS Firefox).

In multiselect mode I am able to focus the selection checkbox and press Space to toggle it.

Reply:
Just not yet implemented (updated the PR description), but should be straightforward.